yurt/yurt_core/setup.py

```diff
@@ -1,20 +1,20 @@
 import os
 import click
 from invoke import run
-from utils import recursive_file_modify, \
-    generate_ssh_keypair, get_project_name_from_repo, add_settings
-from cli import main
+try:
```

Comment:
curious why we're trying both absolute and relative imports here? may need a code comment why